Change function Verb and Add PowerShell 7 ready

JM2K69 opened the pull request:
According to PowerShell Verb I change All verb for the module and I make It fully functional with PowerShell 7

JM2K69 committed:
Change function Verb and Add PowerShell 7 ready

mgreenegit commented:
@gehuang-dut @nitinbps

nitinbps commented on GPRegistryPolicyParser.psm1:
```diff
@@ -184,7 +184,14 @@ Function Parse-PolFile
 
     [string] $policyContents = Get-Content $Path -Raw
-    [byte[]] $policyContentInBytes = Get-Content $Path -Raw -Encoding Byte
+    if ( $PSVersionTable.PSVersion.Major -eq "5")
```
> -eq [](start = 40, length = 4)

would it be -le instead?

JM2K69 replied:
I don't understand what you want sorry ? I don't touch anything rather than the Verb an a switch to make it compatible with Powershell 7

JM2K69 replied:
Yes your write it would be -le instead -eq.

mgreenegit commented:
Wow very good catch. @JM2K69 can you make this small change before we merge?

JM2K69 committed:
$PSVersionTable change

$PSVersionTable.PSVersion.Major -eq "5" to $PSVersionTable.PSVersion.Major -le "5"

JM2K69 commented:
I cr

JM2K69 commented:
I change the $PSVersionTable.PSVersion.Major -eq "5" to $PSVersionTable.PSVersion.Major -le "5" I close this a create an other PR
